scripts/dev.sh

```
@@ -69,129 +184,187 @@ find_free_port() {
+  r) rebuild          remove container node_modules + restart
+  q) quit
+
+EOF
 if [ -n "$LAST" ]; then
-  echo "Or just hit enter to use previous (${PROFILE_NAMES[$LAST]})"
+  echo "Hit enter to repeat last: ${LAST}"
   echo ""
 fi
-read -p "Choice: " choice
+
+read -p "Choice [1-5/d/l/p/c/r/q]: " choice
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)